### PR TITLE
Useful items for new MessageLogger syntax

### PR DESCRIPTION
**Pull request by Dr15Jones**
#### PR description:

- Added a script that can convert old MessageLogger configurations to the new syntax
- Added a 'How To' page for configuration changes people most often apply to the MessageLogger

#### PR validation:

- The script was used to do much of the migration of configurations containing explicit MessageLogger `Service` creations.

### CODE REVIEW

**Comment by Dr15Jones** [comment]
please test

File: FWCore/MessageService/Readme.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+One needs to switch off cerr and switch on cout
+
+```python
+process.load("FWCore.MessageLogger.MessageLogger_cfi")
+process.MessageLogger.cerr.enable = False
+process.MessageLogger.cout.enable = True
+```
+
+## Writing the standard output to a file
+Assing the file you want to write is name `my_file.log` then
```
Comment (makortel) [inline FWCore/MessageService/Readme.md:20]:
```suggestion
Assuming the file you want to write is name `my_file.log` then
```
?

File: FWCore/MessageService/Readme.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# Useful MessageLogger Configuration Changes
```
[the remark this answers is not in the document]
Comment (Dr15Jones) [reply FWCore/MessageService/Readme.md:1]: All configurations already use `FWCore.MessageService` we did that conversion quite a while back.

[the remark this answers is not in the document]
Comment (makortel) [reply FWCore/MessageService/Readme.md:1]: There seem to be many configuration files still referring to `FWCore.MessageLogger`, although there are more using `FWCore.MessageService`
```
$ git grep "FWCore\.MessageLogger\.MessageLogger" | wc -l
760
$ git grep "FWCore\.MessageService\.MessageLogger" | wc -l
1250
```
The header to be included by users is also in `FWCore/MessageLogger`.

A possible workaround could be to add a `README.md` there as well with only an explanatory link to here (can be done after this PR).

**Comment by Dr15Jones** [comment]
please test

**Comment by makortel** [comment]
@cmsbuild, please test

**Comment by makortel** [comment]
+1

**Comment by qliphy** [comment]
+1

**Pull request merged**